public/app/features/dashboard-scene/scene/PanelLinks.tsx

```diff
@@ -11,11 +20,21 @@ interface VizPanelLinksState extends SceneObjectState {
 }
 
 export class VizPanelLinks extends SceneObjectBase<VizPanelLinksState> {
+  protected _variableDependency = new VariableDependencyConfig(this, {
+    onAnyVariableChanged: () => {
+      if (this.state.rawLinks && this.state.rawLinks.length > 0) {
+        this.forceRender();
+      }
+    },
+  });
+
   static Component = VizPanelLinksRenderer;
 }
 
 function VizPanelLinksRenderer({ model }: SceneComponentProps<VizPanelLinks>) {
   const { menu, rawLinks } = model.useState();
+  const dashboard = getDashboardSceneFor(model);
+  sceneGraph.getTimeRange(dashboard).useState();
```

Comment:
why not use `sceneGraph.getTimeRange(model).useState()`  ?

Reply:
didn't think that through, you're right!